README.md

```diff
@@ -2,19 +2,15 @@
 
 Common public api definitions for the DataTrails platform
 
-## Finding and including proto files for depdendecnies
+## Finding and including proto files for dependencies
 
-tools/go.mod is the source of truth for all proto providing dependencies. That file alone specifies both the upstream version we are using *and* is used, via go install, to make the .proto files available locally
+go.mod is the source of truth for all proto providing dependencies. That file alone specifies both the upstream version we are using *and* is used, via go get -tool, to make the .proto files available locally
 
-This corresponds to the practice recommended by grpc-gateway and elsewhere
+This corresponds to the practice recommended by grpc-gateway and elsewhere.
 
-1. *** ALWAYS *** Use tools/go.mod to specify the dependency version.
-2. Add the package to the `go install` command in the apis:preflight task
-3. If necessary, add a var for the new path in any-api **and** then add a reference to that var in the PROTO_INC var.
-
-Following this practice removes the need for dual maintenance of dependency versions in the builder image. It also produces a build cycle that is significantly faster.
-
-Cross repository builds in docker while using go.work to refer to locally modified sources don't work. And this setup is essential for an efficient workflow.
+1. Execute the `go get -tool` command to add the package to the go.mod file.
```

Comment:
Sorry, where do I do this ? the deleted lines tell me how to do this for existing api support. Do we have taskfile support for doing this at the right spot ?

Reply:
The is a manual process that sets up the go.mod file - it should not be executed in any workflow.

Reply:
if we add a new tool the we would exeute the go get -tool command for that tool once
this changes go.mod - after that tool management is transparent

Reply:
Then I think we need a task rune for that please